app/views/splash/index.html.erb

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
- [ ] Please make sure you follow the [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
  * Colors
  * Typography: font face, size and weight.
  * Layout: composition and space between elements.Check these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

Please ensure that the splash screen looks similar to the original design as mentioned in the requirements.

### Your Implementation
![image](https://github.com/emmanuelsteven/BudgetApp/assets/45460192/60ed9d48-baf9-4341-a6ff-44a26e2ab2aa)
### Required
![image](https://github.com/emmanuelsteven/BudgetApp/assets/45460192/29943fb0-fbd6-4d29-9654-0044f0c19efe)